ape_alchemy/provider.py

```diff
@@ -26,6 +27,12 @@ class Alchemy(Web3Provider, UpstreamProvider):
 
     network_uris: Dict[tuple, str] = {}
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.block_page_size = 5000
+        self.concurrency = 1
+        print(f"setting block_page_size to {self.block_page_size} and concurrency to {self.concurrency}")
```

Comment:
I don't think you need to log this, but definitely if you want to print anything, us loggers (as @unparalleled-js said)

Reply:
ah yeah those were there temporarily! let me take them out!